[wpilib] Remove IterativeRobotBase prints

calcmogul opened the pull request:
They don't provide much utility for the end user.

PeterJohnson commented:
I don't really like this change.  I guess the init ones are fine, but the ones in the periodic functions seem pretty important.

calcmogul commented:
OK, I added the periodic messages back in and added a print for when the robot program startup is done, since that's a common unintended use for the init messages.
